src/lib/request.js

```diff
@@ -62,6 +62,11 @@ const request = async (opts) => {
                 if (responseType === ResponseType.Text || !result.length) {
                     result = result.toString();
                 } else if (responseType === ResponseType.JSON) {
+                    const contentType = res.headers['content-type'];
+                    if (!/^application\/json/.test(contentType)) {
```

Comment:
Why not this?
```suggestion
                    if (!/^application\/json$/.test(contentType)) {
```

Reply:
@msk- could be also `Content-type: application/json; charset=utf-8`